lib/KalturaHelpers.php

```diff
@@ -200,8 +200,8 @@ public static function getOption( $name, $default = null ) {
 		$name    = is_string( $name ) ? $name : null;
 		$default = is_bool( $default ) ? $default : null;
 
-		$value = get_option( $name, $default );
-		if ( ! is_null( $value ) ) {
+		$value = self::isPluginNetworkActivated() ? get_site_option( $name, $default ) : get_option( $name, $default );
```

Comment:
The specific instance option should be returned first.
A sub blog can be configured to use a different partner ID. Worth checking that it still works, but this was the behavior in the past.